core_lib/src/util/cameraeasingtype.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CAMERAEASINGTYPE_H
```

Comment:
Also, this file should have a licence header. Though I’m not sure if it’s even needed, seems like one could simply use QEasingCurve::Type directly (But as I said, I can’t do a proper review right no so maybe there’s a justification for this enum that I just haven’t noticed while skimming through the changes. In that case it’s just the missing licence header).

Reply:
I intended to use the QEasingCurve::Type directly, but we ended up with making our own, to make it independent of QT, in case we skipped it. 
I'm fine with you reviewing the code, to get fresh eyes on it, and it's okay if you don't have the time right now, but I have the hope, that it will be reviewed within the next month...

Reply:
Agh.. Didn't mean to resolve it.

The enum has been made because of me, as I don't think it's a good idea to couple the values directly to qt if we can avoid it.

We are extremely coupled to qt though so maybe it was not necessary afterall.

I prefer explicit mappings to our own logic when possible.

Reply:
I've thought more about this, and I think we should use the enum in QEasingCurve, for two reasons:
1) We are already bound to Qt, so it is a minor issue to reinstate an enum, if we sometime in the future want to migrate.
2) There are 41 enums in QEasingCurve, and I use 21 of them. I use the enums 0-16 and 25-28. There are some of the others that could be integrated, but I thought it would be best to start with these 21. If we add some easing-methods later, we MUST add them after the last one, so it doesn't mess up the values in the .pclx files that users have, and rely on.
I see a potential danger here...

Reply:
Added License header.

Reply:
I took a decision. I expanded the enum in cameraeasingtype.h, so all enums in QEasingCurve are listed, and so they correspond to the same int in both enums.
That way, it is easy to add more camerainterpolations if wanted, and there should be no potential danger - if we keep the cameraeasingtype.h enum.